draft-ietf-anima-brski-prm.md

```diff
@@ -2054,103 +2095,109 @@ Within the server log the registrar SHOULD capture this telemetry information.
 
 The following assumes that a Registrar-Agent may need to query the status of a pledge.
 This information may be useful to solve errors, when the pledge was not able to connect to the target domain during the bootstrapping.
-The pledge MAY provide a dedicated endpoint to accept status-requests.
+The pledge MAY provide the dedicated endpoint for the Query Pledge Status operation.
 
 ~~~~ aasvg
-+--------+   +-----------+   +-----------+    +--------+  +---------+
-| Pledge |   | Registrar-|   | Domain    |    | Domain |  | MASA    |
-|        |   | Agent     |   | Registrar |    | CA     |  |         |
-+--------+   +-----------+   +-----------+    +--------+  +---------+
-   |                |                 |              |   Internet |
-   ~                ~                 ~              ~            ~
-   (11) Query Pledge Status
-   ~                ~                 ~              ~            ~
-   |                |                 |              |            |
-   |<--pStatus-Req--|                 |              |            |
-   |--pStatus-Resp->|                 |              |            |
-   |                |                 |              |            |
-   ~                ~                 ~              ~            ~
++--------+    +------------+    +-----------+    +--------+    +------+
+| Pledge |    | Registrar- |    |  Domain   |    | Domain |    | MASA |
+|        |    |   Agent    |    | Registrar |    |   CA   |    |      |
++--------+    +------------+    +-----------+    +--------+    +------+
+ |                  |                 |                 |   Internet |
+ ~                  ~                 ~                 ~            ~
+(11) Query Pledge Status
+ ~                  ~                 ~                 ~            ~
+ |                  |                 |                 |            |
+ |<----opt. TLS---->|                 |                 |            |
+ |<-----qStatus-----|                 |                 |            |
```

Comment:
Brought it closer to PRM defined names, i.e., no "Request"/"Response" as those are problematic for artifact names, which should go over different transports..